scripts/cybershake/add_to_mgmt_queue.py

```diff
@@ -13,7 +13,7 @@
     parser.add_argument(
         "proc_type", type=str, help="The string value of the process type."
     )
-    parser.add_argument("status", type=int, help="The integer value for the status")
+    parser.add_argument("status", type=str, help="The string value of the status")
```

Comment:
Last thing. I'd add the enum as pre-poulated options for this so it errors in case of a misspelling.